consts.js

```diff
@@ -61,7 +61,31 @@ var thirteenStrings = [
     // binary
     "00001101",
     "0b1101",
-
+    // Base 3
+    "111"
```

Comment:
You're inside an array. This line (and all added lines of code) need commas...

Reply:
Awkward